app/javascript/src/components/Dashboard/Contacts/ContactTable/Row.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Checkbox, Dropdown, Avatar, Typography } from "neetoui";
+
+const Row = ({ contact }) => {
+  return (
+    <tr>
+      <td>
+        <Checkbox name={contact.id} />
+      </td>
+
+      <td>
+        <div className="flex items-center space-x-2">
+          <Avatar
+            size={"large"}
+            user={{ name: `${contact.firstName} ${contact.lastName}` }}
+          />
+          <div className="flex flex-col">
+            <Typography style="h5" weight="semi-bold">
+              {contact.firstName} {contact.lastName}
+            </Typography>
+            <Typography style="body3" weight="medium">
+              {contact.role.value}
+            </Typography>
+          </div>
+        </div>
+      </td>
+      <td>{contact.email}</td>
```

Comment:
Please use fixed width and truncate the text beyond that width. Please refer `truncate` classname from tailwindCSS.